src/foam/u2/view/ReciprocalSearch.js

```diff
@@ -104,7 +104,10 @@ foam.CLASS({
         }
 
         return of.getAxiomsByClass(foam.core.Property)
-            .filter((prop) => prop.searchView && ! prop.hidden)
+            .filter((prop) => prop.searchView
+              && ! prop.hidden
+              && ! foam.dao.OneToManyRelationshipProperty.isInstance(prop)
```

Comment:
Rather than hard coding these specific properties, lets update or refine those property types to have "null" as their search view.  This might not forever be the only place we ask a property for its search view, its better that the property be honest about whether it has a suitable search view or not.